docs/src/process/release.md

```diff
@@ -3,28 +3,124 @@
 
 This document outlines the process and key considerations for releasing a new version of the NASA Open MCT project as an NPM (Node Package Manager) package.
 
+## FAQ
+
+1. When do we publish a new version of Open MCT?
+    - At the end of a working sprint (typically) after all blocking issues have been resolved.
+2. Where do we publish?
+    - [NPM](https://www.npmjs.com/package/openmct)
+    - [Github Releases](https://github.com/nasa/openmct/releases)
+2. What do we publish?
+    - What constitutes a "stable" release?
```

Comment:
Shouldn't the different release types be clustered under another header, like 
What kinds of releases do we publish?
- Stable: blah blah
- Latest: The most recently published release.
- Nightly: yada yada